src/helpers/saveResult.js

```diff
@@ -1,6 +1,6 @@
 import fs from 'fs';
 import { MODE_CSV, MODE_SQL } from '../core/outputModes';
-import { outputFileName } from '../../config.json';
+import config from './config';
```

Comment:
As you are exporting a config object, you can leave it here as it was before (`{ outputFileName }`).